go/mysql/replication_position.go

```diff
@@ -120,6 +127,28 @@ func EncodePosition(rp Position) string {
 	return fmt.Sprintf("%s/%s", rp.GTIDSet.Flavor(), rp.GTIDSet.String())
 }
 
+// ParseFilePosition converts a string in the format file:pos
```

Comment:
These changes should go away after we integrate the new flavor.

Reply:
Correct!